static/src/javascripts/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -13,10 +13,10 @@ define([
     function _testABClash(f) {
 
         var contributionsEpic = {name: 'ContributionsEpic20160916', variants: ['control', 'give', 'today', 'make']};
-
-        var contributionsEpicButtons = {name: 'ContributionsEpicButtons20160907', variants: ['control', 'buttons']};
-
-        var clashingTests = [contributionsEpic, contributionsEpicButtons];
+        
```

Comment:
I have rearraged it so it looks like this:

```
var contributionsEpic = {name: 'ContributionsEpic20160916', variants: ['control', 'give', 'today', 'make']};
        var contributionsStory = {name: 'ContributionsStory20160922', variants: ['control', 'story']};
        var clashingTests = [contributionsEpic, contributionsStory];

        return some(clashingTests, function (test) {
            return some(test.variants, function (variant) {
                return f(test.name, variant);
            });
        });
```

Is this ok @NataliaLKB , or do you also want the whitespace between the vars and the return statement removed?